packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -413,6 +433,29 @@ const swipeDown = async ( driver, delay = 3000 ) => {
 	);
 };
 
+// Drag & Drop after element
+const dragAndDropAfterElement = async ( driver, element, nextElement ) => {
+	// Element to drag & drop
+	const elementLocation = await element.getLocation();
+	const elementSize = await element.getSize();
+	const x = elementLocation.x + elementSize.width / 2;
+	const y = elementLocation.y + elementSize.height / 2;
+
+	// Element to drag & drop to
+	const nextElementLocation = await nextElement.getLocation();
+	const nextElementSize = await nextElement.getSize();
+	const nextYPosition = isAndroid()
+		? elementLocation.y + nextElementLocation.y + nextElementSize.height
+		: nextElementLocation.y + nextElementSize.height;
+
+	const action = new wd.TouchAction( driver )
+		.press( { x, y } )
```

Comment:
I ended up using `press` since `longPress` was not triggering correctly for the Drag & Drop feature on iOS,